### PR TITLE
Fixed deep-foundation/deepcase#100

### LINKED ISSUE #100: Bug: When Insert frame is opened new Insert clicks reload that frame in old position
I found bug into placement behavior: If InsertCard frame still opened, new Insert command not moved this onto click position. 
That conditioned permanent undefinite state of "cy" variable ( _see imports/cyto/hooks.tsx line 199 for details_ ).

**Frist:** open Insert menu here
изображение
**Second:** want to reopen Inser menu here
изображение
**Third:** menu reloaded at first position instead of new position
изображение

### PR DESCRIPTION
**Pull request by agsagds**
Bug: https://github.com/deep-foundation/deepcase/issues/100
Replace all "cy" object closures at "cyRef" closures which works correctly.

@Konard You are welcome to review!

### CODE REVIEW

File: imports/cyto/graph.tsx [hunk 1]
```diff
@@ -132,7 +132,7 @@ export default function CytoGraph({
           pan={cytoViewportRef?.current?.value?.pan}
           zoom={cytoViewportRef?.current?.value?.zoom}
           style={ { width: '100%', height: '100vh' } }
-        />
+        />
```
Comment (konard) [inline imports/cyto/graph.tsx]:
```suggestion
        />
```

File: imports/cyto/graph.tsx [hunk 2]
```diff
@@ -142,7 +142,7 @@ export default function CytoGraph({
       <Text position="fixed" left={0} bottom={0} p={4}>
         {pckg.version}
       </Text>
-    </div>
+    </div>
```
Comment (konard) [inline imports/cyto/graph.tsx]:
```suggestion
    </div>
```

**Review by konard — CHANGES_REQUESTED** [review]

**Review by konard — APPROVED** [review]

**Comment by FreePhoenix888** [comment]
@Konard , what about 5000 rubles?

**Comment by konard** [comment]
@FreePhoenix888 we wait few days and see how Deep.Case behaves. If nothing else breaks the reward will be paid.

**Comment by konard** [comment]
5000 rub reward is paid

**Pull request merged**